testsuite/classlibrary/TestSoundFile.sc

```diff
@@ -0,0 +1,51 @@
+TestSoundFile : UnitTest {
+
+	var server, soundFile, path, info;
+
+	setUp {
+
+		path = Platform.resourceDir +/+ "sounds/a11wlk01.wav";
+		info = IdentityDictionary[
+			(\numFrames -> 188893),
+			(\numChannels -> 1),
+			(\path -> path),
+			(\sampleRate -> 44100),
+			(\sampleFormat -> "int16"),
+			(\headerFormat -> "WAV")
+		];
+		soundFile = SoundFile(path);
+
+	}
+
+	tearDown {
+
+		soundFile.close;
+
+	}
```

Comment:
the extra whitespace is fine elsewhere but in cases like this it starts to become noticeable. in general i think you should stick with existing convention and avoid adding extra newlines after the opening brace and before closing brace of a SC function.

Reply:
not asking you to change it, just making a note for future work

Reply:
ok, I thought it was a best practice to add a space before and after. But, there's not much going on in these tests so it does look strange. Thanks for the detailed review!